doc/github_handler.rst

```diff
@@ -0,0 +1,60 @@
+GitHub Handler
+==============
+
+GitHub Handler is a Universum mode that serves as GitHub Application, helping to perform and report checks on
+new commits to a repository. In this mode Universum can create new check runs on GitHub and trigger an already
+set up automation server to perform these checks. GitHub Handler parses all required params and passes them
+to the triggered builds.
+
+
+How to set up GitHub auto check using Unversum
+----------------------------------------------
+
+Universum in default, 'main' mode when set up to ``--vcs-type=github`` and with ``--report-to-review`` passed
+as parameter, can post `check run` statuses to GitHub to be depicted on 'Checks' page in pull requests or as
+a simple icon near any checked commit. To do so, it needs `a set of parameters <args.html#GitHub>`__
+to be passed to it.
+
+.. note::
+
+    During these checks, if some steps were using :doc:`Universum analyzers <code_report>`,
+    and any issues are found, they will also be reported in comments.
+
+These required parameters are sent by GitHub via web-hooks after the reuired `check run` is created. To do so,
```

Comment:
@k-dovgan, could you, please confirm all of these parameters are provided when using GitHub Handler?
If yes, please change the wording to emphasize this.
If no, please provide the exact list of parameters.